backend/core/auth.py

```
@@ -0,0 +1,138 @@
+        
+        # Return user info with role
+        return {
+            "id": user.id,
+            "email": user.email,
+            "role": role,
+            "school_id": school_id
+        }
+        
+    except HTTPException:
+        raise
+    except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)